File: lab-2/src/pokemons/Carracosta.java
package pokemons;

import moves.*;
import ru.ifmo.se.pokemon.*;

public class Carracosta extends Tirtouga{
    public Carracosta(String name, int level) {
        super(name, level);
        setStats(74,108,133,83,65,32);
        setType(Type.WATER, Type.ROCK);
        addMove(new CrossChop());
    }
}

File: lab-2-kotlin/src/pokemons/Carracosta.kt
package pokemons

import moves.CrossChop

import ru.ifmo.se.pokemon.*

class Carracosta(name: String?, level: Int) : Tirtouga(name, level) {
    init {
        setStats(74.0, 108.0, 133.0, 83.0, 65.0, 32.0)
        setType(Type.WATER, Type.ROCK)
        addMove(CrossChop())
    }
}